prisma/migrations/20210828181750_updated/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  Warnings:
```

Comment:
**Question**

Will these migrations will be consolidated prior to merge?

Reply:
Yes. I will make one single migration.